lib/jwt/taskrouter/util.js

```diff
@@ -65,7 +66,99 @@ function defaultEventBridgePolicies(accountSid, channelId) {
   ];
 }
 
+/**
+ * Generate TaskRouter workspace url
+ *
+ * @param {string} [workspaceSid] workspace sid or '**' for all workspaces
```

Comment:
Ah it looks like that will produce different behavior though, I guess thats what we want though. carry on.